### DISCUSSION TITLE
Using Markdown in theme extension and block overriding

### DISCUSSION THREAD

**Discussion by joapuiib** [squidfunk/mkdocs-material#7647] [Q&A] [answered]
I want to override the `announce` block Material for Mkdocs in order to show a announcement in the top of the page.

Is it possible to use Markdown in template overriding?

```html
{% block announce %}
<div class="under-construction" markdown>

:construction: :construction_site: Site under construction! :construction: :construction_site:

</div>
{% endblock %}
```

I'm assuming it doesn't work.

Also, the example in this repository uses HTML directly:
https://github.com/squidfunk/mkdocs-material/blob/1357cd26af7cbbe9dd6e3b3a355feba8b5b2999d/material/overrides/main.html#L8-L23

I just want to know if it's possible.
Thanks in advance!

**Comment by squidfunk** [comment, marked as answer]
No, it's not possible, since it's too late in the rendering process – all Markdown has been translated to HTML and Jinja templates only understand HTML, so any Markdown will remain verbatim. It might be doable with a plugin or hook, though.

**Reply by joapuiib** [reply]
Thanks for clarifying!
